test/complexity/measures/statistical_complexity.jl

```diff
@@ -0,0 +1,40 @@
+using ComplexityMeasures, Distances
+using Test
+
+@testset "Statistical Complexity" begin
+
+    m, τ = 6, 1
+    x = randn(10000)
+    c = StatisticalComplexity(
+        dist=JSDivergence(),
+        est=SymbolicPermutation(; m, τ),
+        entr=Renyi()
+    )
+    # complexity of white noise should be very close to zero
+    compl = complexity(c, x)
+    @test compl isa Real
+    @test 0 < compl < 0.02
+    # complexity of monotonically ascending signal should be exactly zero
+    compl = complexity(c, collect(1:100))
+    @test compl == 0.0
+
+    # test the wrapper
+    entr, compl = entropy_complexity(c, x)
+    @test compl isa Real
+    @test entr isa Real
```

Comment:
These two test aren't useful. You are anwyasy testing if they are real since you use `< 0`.